src/api-service/__app__/onefuzzlib/agent_events.py

```diff
@@ -76,6 +77,8 @@ def on_state_update(
         # successfully.
         node.reimage_requested = False
         node.set_state(state)
+        node.created_at = datetime.datetime.utcnow()
```

Comment:
i don't think `created_at` should be mutable. The node is created once and reimage multiple times. 
I propose rename this to last_os_deployment or something similar to avoid the confusion.

Reply:
I think that's fair! Made the change.